data-lineage/src/main/java/com/wipro/ats/bdre/lineage/entiity/Table.java

```diff
@@ -43,7 +43,7 @@
 
 	public Table(String tableName, String dataBase, String alias, EntityType entityType, boolean isDatabaseTable) throws ETLException {
 		// eliminate "." from table name
-		super((tableName.split("\\."))[tableName.split("\\.").length-1], "Table", "record", tableName);
+		super((tableName.split("\\."))[tableName.split("\\.").length-1], "Table", "records", tableName);
 //		++counter;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)